temporal-serviceclient/src/test/java/io/temporal/serviceclient/HealthCheckTest.java

```diff
@@ -30,33 +30,17 @@
 
   private static final boolean useDockerService =
       Boolean.parseBoolean(System.getenv("USE_DOCKER_SERVICE"));
-  private static final String HEALTH_CHECK_SERVICE_NAME =
-      "temporal.api.workflowservice.v1.WorkflowService";
 
   @Test
   public void testHealthCheck() {
     if (useDockerService) {
       try {
-        WorkflowServiceStubsImpl service =
-            (WorkflowServiceStubsImpl) WorkflowServiceStubs.newInstance();
-        service.checkHealth(HEALTH_CHECK_SERVICE_NAME);
+        // Stub creation triggers health check by default, unless disableHealthCheck flag is set in
+        // the WorkflowServiceStubsOptions.
+        WorkflowServiceStubs.newInstance();
       } catch (Exception e) {
         Assert.fail("Health check failed");
       }
     }
   }
-
-  @Test
```

Comment:
Can we save this test though?
Maybe using the `checkHealth(Channel channel, String serviceName)`

Reply:
I don't like this ninja parameter passing, I thought that we should either make service name configurable via service stub options, or not have this test at all. And I didn't see a use case for making this parameter configurable, hence I've decided to delete the test. Note that old test was actually doing health check twice (first during init, then explicitly), which was slightly confusing.